### PR TITLE
More comments in the signalling code

### PR DESCRIPTION
**Pull request by RoxyFarhad**

### CODE REVIEW

**Review by vijayvuyyuru — APPROVED** [review]
These changes look good and will definetly help in the future if someone else starts looking in here. I can publish a new release after the metric change is merged in! Nice work!

**Comment by ohEmily** [comment]
[minor] could you add your name to your comments to follow Eric's convention? I know people can use git blame but sometimes these are harder to follow.
```
// Note(erd): 
```

File: rpc/wrtc_call_queue_mongodb.go
Comment (ohEmily) [inline rpc/wrtc_call_queue_mongodb.go:427]: [minor] would it be worth renaming to `initialized`? or something else that documents that it only happens once.

File: rpc/wrtc_call_queue_mongodb.go
Comment (ohEmily) [inline rpc/wrtc_call_queue_mongodb.go:607]: would it be worth adding a `debug` log here or would it be so insanely noisy that it's not useful?

File: rpc/wrtc_call_queue_mongodb.go
Comment (ohEmily) [inline rpc/wrtc_call_queue_mongodb.go:628]: [q] wouldn't this log once per channel without including identifying information for the channel? Would it make sense to instead log the event in the log line above at line 619?

(Apologies if I'm misunderstanding here!)

File: rpc/wrtc_call_queue_mongodb.go
Comment (RoxyFarhad) [reply rpc/wrtc_call_queue_mongodb.go:607]: i think it would be quite noisy and not useful

File: rpc/wrtc_call_queue_mongodb.go
Comment (RoxyFarhad) [reply rpc/wrtc_call_queue_mongodb.go:628]: I actually just think we should remove this here, it will be noisy and it was mainly for me debugging locally :)

File: rpc/wrtc_call_queue_mongodb.go
Comment (RoxyFarhad) [reply rpc/wrtc_call_queue_mongodb.go:427]: changed to isInitialized

File: .vscode/launch.json
Comment (ohEmily) [inline .vscode/launch.json:1]: (maybe committed by accident?)

File: rpc/wrtc_call_queue_mongodb.go
Comment (ohEmily) [reply rpc/wrtc_call_queue_mongodb.go:607]: thanks!

**Review by ohEmily — APPROVED** [review]
LGTM! Wasn't sure exactly how you decided which comments to prefix with `note(roxy)` but it's a detail to add that; happy to defer to your decision tree. Thank you SO MUCH for marking this up and working to improve our signaling knowledge.

**Review by ohEmily — APPROVED** [review]

**Pull request merged**
